packages/cms-cli/bin/cli.js

```diff
@@ -27,10 +28,16 @@ const listCommand = require('../commands/list');
 const openCommand = require('../commands/open');
 const mvCommand = require('../commands/mv');
 
+// Point update-notifier towards @hubspot/cli
+const pkg = { name: '@hubspot/cli', version };
+
 const notifier = updateNotifier({ pkg });
 
 notifier.notify({
   shouldNotifyInNpmScript: true,
+  message:
+    chalk.bold('The CMS CLI is now the HubSpot CLI') +
+    '\n\nTo upgrade, run:\n\nnpm uninstall -g @hubspot/cms-cli\nand npm install -g @hubspot/cli',
```

Comment:
What if we adjust the logic such that we're detecting that this code is part of `@hubspot/cms-cli` so that we can merge the changes back into `master` so that the process of merging changes from `cms-cli` branch back into `master` is straightforward?

Reply:
That's a good point, updated